docs/element-definitions.md

```diff
@@ -1139,6 +1139,10 @@ The Resource representation with its available transitions and its data.
 
         The `content` MUST NOT include more than one `Data Structure`.
 
+#### Classifications
```

Comment:
We should add a `hosts` attribute above, and also to the `Transition` element below.

```markdown
- `hosts` ([Array][][[Resource][]]) - Resources in the hosts attribute SHOULD implicitly be treated as they have the `host` resource classification. _See `host` classification in [Resource][] for further semantics._ If `hosts` attribute is specified at the Root, Resource, or Transition level, it will be overidden by this value.
```

The example for this element can look like:

```yaml
element: resource
attributes:
  href: /foo
  hosts:
    element: array
    content:
      - element: resource
        attributes:
          href: {scheme}://example.com/
          hrefVariables:
            element: hrefVariables
            content:
              - element: member
                content:
                  key: scheme
                  value:
                    element: enum
                    attributes:
                      enumerations:
                        - http
                        - https
```